airbyte-config/config-models/src/main/resources/types/ConnectorJobOutput.yaml

```diff
@@ -12,12 +12,12 @@ properties:
     type: string
     enum:
       - checkConnection
-      - discoverCatalog
+      - discoverCatalogId
       - spec
   checkConnection:
     "$ref": StandardCheckConnectionOutput.yaml
-  discoverCatalog:
-    existingJavaType: io.airbyte.protocol.models.AirbyteCatalog
+  discoverCatalogId:
+    type: string
```

Comment:
Can we add a description to this, explaining how we determine the ID, if it's a UUID or not, etc?

Reply:
Realized that maybe we can include a `format: uuid` here to get better autogenerated types throughout? Would be great to be able to say `SynchronousResponse<UUID>` instead of `SynchronousResponse<String>`!

Reply:
Good call, done!